components/hab/src/command/bldr/job/start.rs

```diff
@@ -29,6 +29,8 @@ pub fn start(ui: &mut UI,
                 ui.warn(rdep.to_string())?;
             }
 
+            ui.warn("Note: dependencies from private origins are omitted for non-members.");
+
             let question = "If you choose to start a group build for this package, all of the \
```

Comment:
Do you mean:
```suggestion
            let question = "Starting a group build for this package will \ also build all of the reverse dependencies listed above. Is this what you want?";
```
You could also add the reason they might not want to do it, which I assume is because it is time-consuming. Does it have any other impact?

Reply:
Right, builds can take time and tie up resources that a user might not want. So having a basic confirm message seems appropriate.